frontend/src/modules/Survey/Components/StepBegin.tsx

```diff
@@ -98,48 +46,39 @@ export const StepBegin = ({
           <StyledWelcomeText>Find study partners!</StyledWelcomeText>
         </StyledTitleWrapper>
         <StyledFields>
-          <StyledTextFieldWrapper
-            style={{
-              marginBottom: calculatePadding(errorEnum.NAME) + 'rem',
-            }}
-          >
-            <NameField
-              containerStyle={textContainerStyle}
-              inputStyle={nameTextInputStyle}
-              value={name}
-              onChange={(e: React.ChangeEvent<HTMLInputElement>) =>
-                setName(e.target.value)
-              }
-              // TODO: add extra thing in conditional so that when they type something that is valid, the error goes away
-              error={
-                error === errorEnum.NAME || error === errorEnum.BOTH
-                  ? error
-                  : ''
-              }
-            />
-          </StyledTextFieldWrapper>
-          <StyledTextFieldWrapper
-            style={{
-              marginBottom: calculatePadding(errorEnum.EMAIL) + 'rem',
-            }}
-          >
-            <EmailField
-              containerStyle={textContainerStyle}
-              inputStyle={emailTextInputStyle}
-              value={email}
-              onChange={(e: React.ChangeEvent<HTMLInputElement>) =>
-                setEmail(e.target.value)
-              }
-              error={
-                error === errorEnum.EMAIL || error === errorEnum.BOTH
-                  ? error
-                  : ''
-              }
-            />
-          </StyledTextFieldWrapper>
+          <TextField
+            placeholder="Name"
+            variant="standard"
+            sx={textInputStyle}
+            value={name}
+            onChange={(e: React.ChangeEvent<HTMLInputElement>) =>
+              setName(e.target.value)
+            }
+          />
+          <TextField
+            value={email}
+            sx={textInputStyle}
+            variant="standard"
+            placeholder="Cornell NetID"
+            onChange={(e: React.ChangeEvent<HTMLInputElement>) =>
+              setEmail(e.target.value)
+            }
+          />
         </StyledFields>
-        <Box sx={{ marginLeft: 'auto', textAlign: 'center' }}>
-          <IconButton className="next" onClick={handleNext}>
+        <Box
+          sx={{
+            marginLeft: 'auto',
+            textAlign: 'center',
```

Comment:
Probably a more elegant solution now is to use a column-oriented flexbox to group the button and "Next" instead of using a `<br>` element (bad use of semantic HTML here)